### PR TITLE
CppCheck HVAC VRF

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes programming syntax
 - Uses CppCheck as guide

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 1]
```diff
@@ -174,13 +174,6 @@ void SimulateVRF(EnergyPlusData &state,
         }
         CompIndex = VRFTUNum;
 
-        // suppress unused warnings temporarily until VRF inherits HVACSystemData
-        if (OAUnitNum > 0) {
-            bool tmpFlag = false;
-            if (OAUCoilOutTemp > 0.0) tmpFlag = true;
-            if (ZoneEquipment) tmpFlag = true;
-        }
-
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:183]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:179]:(style),[unreadVariable],Variable 'tmpFlag' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:180]:(style),[unreadVariable],Variable 'tmpFlag' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:181]:(style),[unreadVariable],Variable 'tmpFlag' is assigned a value that is never used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 2]
```diff
@@ -363,10 +356,7 @@ void CalcVRFCondenser(EnergyPlusData &state, int const VRFCond)
 
     static constexpr std::string_view RoutineName("VRFCondenser");
 
-    int NumTU;         // loop counter
-    int TUIndex;       // Index to terminal unit
-    int CoolCoilIndex; // index to cooling coil in terminal unit
-    int HeatCoilIndex; // index to heating coil in terminal unit
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:369]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:367]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:368]:(style),[variableScope],The scope of the variable 'CoolCoilIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:369]:(style),[variableScope],The scope of the variable 'HeatCoilIndex' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 3]
```diff
@@ -399,11 +388,9 @@ void CalcVRFCondenser(EnergyPlusData &state, int const VRFCond)
     Real64 InputPowerMultiplier;      // Multiplier for power when system is in defrost
     Real64 LoadDueToDefrost;          // Additional load due to defrost
     Real64 DefrostEIRTempModFac;      // EIR modifier for defrost (function of entering drybulb, outside wetbulb)
-    int HRCAPFT;                      // index to heat recovery CAPFTCool curve
     Real64 HRCAPFTConst;              // stead-state capacity fraction
     Real64 HRInitialCapFrac;          // Fractional cooling degradation at the start of heat recovery from cooling mode
     Real64 HRCapTC;                   // Time constant used to recover from initial degradation in cooling heat recovery
-    int HREIRFT;                      // Index to cool EIR as a function of temperature curve for heat recovery
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:406]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:386]:(style),[variableScope],The scope of the variable 'Stage' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:402]:(style),[variableScope],The scope of the variable 'HRCAPFT' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:406]:(style),[variableScope],The scope of the variable 'HREIRFT' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 4]
```diff
@@ -3271,8 +3258,6 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
     for (int VRFTUNum = 1; VRFTUNum <= state.dataHVACVarRefFlow->NumVRFTU; ++VRFTUNum) {
 
         //     initialize local node number variables
-        int FanInletNodeNum = 0;
-        int FanOutletNodeNum = 0;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:3275]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:3274]:(style),[variableScope],The scope of the variable 'FanInletNodeNum' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:3275]:(style),[variableScope],The scope of the variable 'FanOutletNodeNum' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 5]
```diff
@@ -7510,14 +7495,9 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
...
 
     auto &CheckVRFCombinationRatio = state.dataHVACVarRefFlow->CheckVRFCombinationRatio;
-    bool FoundAll;                      // temporary variable used to check all terminal units
-    bool errFlag;                       // temporary variable used for error checking
     Real64 TUCoolingCapacity;           // total terminal unit cooling capacity
     Real64 TUHeatingCapacity;           // total terminal unit heating capacity
     int VRFCond;                        // index to VRF condenser
-    int TUListNum;                      // index to terminal unit list
-    int TUIndex;                        // index to terminal unit
-    int NumTU;                          // DO Loop index counter
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7520]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7513]:(style),[variableScope],The scope of the variable 'FoundAll' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7514]:(style),[variableScope],The scope of the variable 'errFlag' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7518]:(style),[variableScope],The scope of the variable 'TUListNum' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7519]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7520]:(style),[variableScope],The scope of the variable 'NumTU' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 6]
```diff
@@ -4346,7 +4331,7 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
             vrfTU.DesignSpecMultispeedHPType = cAlphaArgs(20);
             vrfTU.DesignSpecMultispeedHPName = cAlphaArgs(21);
             vrfTU.DesignSpecMSHPIndex = UnitarySystems::getDesignSpecMSHPIndex(state, cAlphaArgs(21));
-            auto &designSpecFan = state.dataUnitarySystems->designSpecMSHP[vrfTU.DesignSpecMSHPIndex];
+            auto const &designSpecFan = state.dataUnitarySystems->designSpecMSHP[vrfTU.DesignSpecMSHPIndex];
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:4334]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:4349]:(style),[constVariable],Variable 'designSpecFan' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 7]
```diff
@@ -5256,7 +5241,7 @@ void CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum, bool &
 {
 
     constexpr static std::string_view cTerminalUnitType("ZoneHVAC:TerminalUnit:VariableRefrigerantFlow");
-    auto &nodeID = state.dataLoopNodes->NodeID;
+    auto const &nodeID = state.dataLoopNodes->NodeID;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:5244]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:5259]:(style),[constVariable],Variable 'nodeID' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 8]
```diff
@@ -7551,14 +7531,7 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
...
     Real64 TempSize;          // autosized value of coil input field
     int FieldNum = 2;         // IDD numeric field number where input field description is found
-    int SizingMethod;       // Integer representation of sizing method name (e.g., CoolingAirflowSizing, HeatingAirflowSizing, CoolingCapacitySizing,
-                            // HeatingCapacitySizing, etc.)
-    bool PrintFlag = true;  // TRUE when sizing information is reported in the eio file
-    int zoneHVACIndex;      // index of zoneHVAC equipment sizing specification
-    int SAFMethod(0);       // supply air flow rate sizing method (SupplyAirFlowRate, FlowPerFloorArea, FractionOfAutosizedCoolingAirflow,
-                            // FractionOfAutosizedHeatingAirflow ...)
-    int CapSizingMethod(0); // capacity sizing methods (HeatingDesignCapacity, CapacityPerFloorArea, FractionOfAutosizedCoolingCapacity, and
-                            // FractionOfAutosizedHeatingCapacity )
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7561]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7558]:(style),[unreadVariable],Variable 'SAFMethod' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7560]:(style),[unreadVariable],Variable 'CapSizingMethod' is assigned a value that is never used.

[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7558]:(style),[variableScope],The scope of the variable 'SAFMethod' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7560]:(style),[variableScope],The scope of the variable 'CapSizingMethod' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 9]
```diff
@@ -7786,28 +7762,28 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
...
+            SizingMethod = HeatingCapacitySizing; // either this isn't needed or needs to be assigned to EqSizing
             TempSize = AutoSize;
             PrintFlag = false;
             state.dataSize->DataScalableSizingON = true;
             state.dataSize->DataFlowUsedForSizing = state.dataSize->FinalZoneSizing(state.dataSize->CurZoneEqNum).DesHeatVolFlow;
             if (state.dataSize->ZoneHVACSizing(zoneHVACIndex).HeatingCapMethod == FractionOfAutosizedHeatingCapacity) {
                 state.dataSize->DataFracOfAutosizedHeatingCapacity = state.dataSize->ZoneHVACSizing(zoneHVACIndex).ScaledHeatingCapacity;
             }
-            bool errorsFound = false;
+            errorsFound = false;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7780]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7789]:(style),[shadowVariable],Local variable 'errorsFound' shadows outer variable
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7804]:(style),[shadowVariable],Local variable 'errorsFound' shadows outer variable
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7930]:(style),[shadowVariable],Local variable 'errorsFound' shadows outer variable

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 10]
```diff
@@ -7708,7 +7684,7 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
             state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxCoolAirVolFlow = sizingCoolingAirFlow.size(state, TempSize, errorsFound);
 
         } else if (SAFMethod == FlowPerCoolingCapacity) {
-            SizingMethod = CoolingCapacitySizing;
+            SizingMethod = CoolingCapacitySizing; // either this isn't needed or needs to be assigned to EqSizing
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7687]: Usually this equivalence would be followed by setting EqSizing so either EqSizing is missing (need to check sizing calls) or this line can be deleted. So left it here as a reminder.

        EqSizing.SizingMethod(SizingMethod) = SAFMethod;

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 11]
```diff
@@ -8896,19 +8873,16 @@ void VRFCondenserEquipment::SizeVRFCondenser(EnergyPlusData &state)
 
     static constexpr std::string_view RoutineName("SizeVRFCondenser");
 
-    int PltSizCondNum;         // Plant Sizing index for condenser loop
     Real64 rho;                // local fluid density [kg/m3]
     Real64 Cp;                 // local fluid specific heat [J/kg-k]
     Real64 tmpCondVolFlowRate; // local condenser design volume flow rate [m3/s]
-    bool ErrorsFound;          // indicates problem with sizing
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:8903]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:8899]:(style),[variableScope],The scope of the variable 'PltSizCondNum' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:8903]:(style),[variableScope],The scope of the variable 'ErrorsFound' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 12]
```diff
@@ -9076,10 +9050,8 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
     int VRFCond = this->VRFSysNum;
     Real64 FullOutput = 0.0;   // unit full output when compressor is operating [W]
     Real64 TempOutput = 0.0;   // unit output when iteration limit exceeded [W]
-    int SolFla = 0;            // Flag of RegulaFalsi solver
     Real64 TempMinPLR = 0.0;   // min PLR used in Regula Falsi call
     Real64 TempMaxPLR = 0.0;   // max PLR used in Regula Falsi call
-    bool ContinueIter;         // used when convergence is an issue
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9082]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9079]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9082]:(style),[variableScope],The scope of the variable 'ContinueIter' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 13]
```diff
@@ -9153,7 +9125,7 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
     // set supplemental heating coil calculation if the condition requires
     if (this->SuppHeatingCoilPresent) {
         if (this->isSetPointControlled) {
-            auto &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
+            auto const &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9128]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9156]:(style),[constVariable],Variable 'thisSuppHeatCoilAirInletNode' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 14]
```diff
@@ -9650,8 +9621,6 @@ void ReportVRFTerminalUnit(EnergyPlusData &state, int const VRFTUNum) // index t
     bool HRHeatRequestFlag;      // - indicates TU could be in heat mode
     bool HRCoolRequestFlag;      // - indicates TU could be in cool mode
 
-    DXCoolingCoilIndex = state.dataHVACVarRefFlow->VRFTU(VRFTUNum).CoolCoilIndex;
-    DXHeatingCoilIndex = state.dataHVACVarRefFlow->VRFTU(VRFTUNum).HeatCoilIndex;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9654]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9653]:(style),[unreadVariable],Variable 'DXCoolingCoilIndex' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9654]:(style),[unreadVariable],Variable 'DXHeatingCoilIndex' is assigned a value that is never used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 15]
```diff
@@ -10035,8 +10004,6 @@ void InitializeOperatingMode(EnergyPlusData &state,
     Real64 SPTempHi;         // thermostat setpoint high
     Real64 SPTempLo;         // thermostat setpoint low
     int NumTU;               // loop counter, number of TU's in list
-    int TUIndex;             // index to TU
-    int ThisZoneNum;         // index to zone number where TU is located
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10039]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10038]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10039]:(style),[variableScope],The scope of the variable 'ThisZoneNum' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 16]
```diff
@@ -10182,12 +10148,13 @@ void InitializeOperatingMode(EnergyPlusData &state,
...
             //     more logic may be needed here, what is the OA flow rate, was last mode heating or cooling, what control is used, etc...
 
+            int ThisZoneNum = state.dataHVACVarRefFlow->VRFTU(TUIndex).ZoneNum;
             getVRFTUZoneLoad(state, TUIndex, ZoneLoad, LoadToHeatingSP, LoadToCoolingSP, true);
 
             if (state.dataHVACVarRefFlow->VRF(VRFCond).ThermostatPriority == ThermostatCtrlType::ThermostatOffsetPriority) {
                 //         for TSTATPriority, just check difference between zone temp and thermostat setpoint
                 if (ThisZoneNum > 0) {
-                    auto &thisZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ThisZoneNum);
+                    auto const &thisZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ThisZoneNum);
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10157]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10190]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 17]
```diff
@@ -10844,7 +10811,7 @@ void getVRFTUZoneLoad(
 bool getVRFTUNodeNumber(EnergyPlusData &state, int const nodeNumber)
 {
     for (int vrfTUIndex = 1; vrfTUIndex <= state.dataHVACVarRefFlow->NumVRFTU; ++vrfTUIndex) {
-        auto &vrfTU = state.dataHVACVarRefFlow->VRFTU(vrfTUIndex);
+        auto const &vrfTU = state.dataHVACVarRefFlow->VRFTU(vrfTUIndex);
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10814]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10847]:(style),[constVariable],Variable 'vrfTU' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 18]
```diff
@@ -10975,7 +10941,6 @@ void VRFTerminalUnitEquipment::CalcVRFIUVariableTeTc(EnergyPlusData &state,
     // Get the equipment/zone index corresponding to the VRFTU
     CoolCoilNum = this->CoolCoilIndex;
     HeatCoilNum = this->HeatCoilIndex;
-    ZoneIndex = this->ZoneNum;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10978]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10978]:(style),[unreadVariable],Variable 'ZoneIndex' is assigned a value that is never used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 19]
```diff
@@ -11351,8 +11316,6 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         // Evaporator (IU side) operational parameters
         Pevap = this->refrig->getSatPressure(state, this->IUEvaporatingTemp, RoutineName);
         Psuction = Pevap;
-        Tsuction = this->IUEvaporatingTemp; // GetSatTemperatureRefrig(state,  this->RefrigerantName, max( min( Psuction, RefPHigh ), RefPLow ),
-                                            // RefrigerantIndex, RoutineName );
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11355]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11444]:(style),[redundantAssignment],Variable 'Tsuction' is reassigned a value before the old one has been used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 20]
```diff
@@ -11793,7 +11756,6 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         // Evaporator (IU side) operational parameters
         Pevap = this->refrig->getSatPressure(state, this->IUEvaporatingTemp, RoutineName);
         Psuction = Pevap;
-        Tsuction = this->IUEvaporatingTemp;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11796]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11917]:(style),[redundantAssignment],Variable 'Tsuction' is reassigned a value before the old one has been used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 21]
```diff
@@ -12421,10 +12383,8 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
     Real64 FullOutput;     // unit full output when compressor is operating [W]
     Real64 TempOutput;     // unit output when iteration limit exceeded [W]
     Real64 NoCompOutput;   // output when no active compressor [W]
-    int SolFla;            // Flag of RegulaFalsi solver
     Real64 TempMinPLR;     // min PLR used in Regula Falsi call
     Real64 TempMaxPLR;     // max PLR used in Regula Falsi call
-    bool ContinueIter;     // used when convergence is an issue
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12427]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12424]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12427]:(style),[variableScope],The scope of the variable 'ContinueIter' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 22]
```diff
@@ -12493,18 +12453,18 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
...
-        auto &thisAirInNode = state.dataLoopNodes->Node(state.dataDXCoils->DXCoil(this->HeatCoilIndex).AirInNode);
+        auto const &thisAirInNode = state.dataLoopNodes->Node(state.dataDXCoils->DXCoil(this->HeatCoilIndex).AirInNode);
         this->coilInNodeT = thisAirInNode.Temp;
         this->coilInNodeW = thisAirInNode.HumRat;
     }
 
     // set supplemental heating coil calculation if the condition requires
     if (this->SuppHeatingCoilPresent) {
-        auto &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
+        auto const &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12467]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12496]:(style),[constVariable],Variable 'thisAirInNode' can be declared as reference to const
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12500]:(style),[constVariable],Variable 'thisAirInNode' can be declared as reference to const
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12507]:(style),[constVariable],Variable 'thisSuppHeatCoilAirInletNode' can be declared as reference to const

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 23]
```diff
@@ -13004,7 +12965,6 @@ Real64 VRFTerminalUnitEquipment::CalVRFTUAirFlowRate_FluidTCtrl(EnergyPlusData &
         using SingleDuct::SimATMixer;
 
         int constexpr Mode(1);  // Performance mode for MultiMode DX coil. Always 1 for other coil types
-        int OAMixNode;          // index to the mix node of OA mixer
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13007]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13007]:(style),[variableScope],The scope of the variable 'OAMixNode' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 24]
```diff
@@ -13684,8 +13642,6 @@ void VRFCondenserEquipment::VRFOU_CompSpd(
     static constexpr std::string_view RoutineName("VRFOU_CompSpd");
 
     // variable initializations: component index
-    TUListNum = this->ZoneTUListPtr;
-    NumTUInList = state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).NumTUInList;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13688]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13688]:(style),[unreadVariable],Variable 'NumTUInList' is assigned a value that is never used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 25]
```diff
@@ -13825,8 +13779,6 @@ void VRFCondenserEquipment::VRFOU_CompCap(
     static constexpr std::string_view RoutineName("VRFOU_CompCap");
 
     // variable initializations: component index
-    TUListNum = this->ZoneTUListPtr;
-    NumTUInList = state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).NumTUInList;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13829]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13829]:(style),[unreadVariable],Variable 'NumTUInList' is assigned a value that is never used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 26]
```diff
@@ -14031,10 +13983,7 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                 Real64 CondHeat = Q_evap_req * C_cap_operation0 / this->RatedEvapCapacity; // 150130 To be confirmed
                 int CAPFT = this->OUCoolingCAPFT(CounterCompSpdTemp);
 
-                // Update Te' (SmallLoadTe) to meet the required evaporator capacity
-                MinOutdoorUnitTe = 6;
                 P_discharge = this->refrig->getSatPressure(state, T_discharge, RoutineName);
-
                 MinRefriPe = this->refrig->getSatPressure(state, -15, RoutineName);
                 MinOutdoorUnitPe = max(P_discharge - this->CompMaxDeltaP, MinRefriPe);
                 MinOutdoorUnitTe = this->refrig->getSatTemperature(state, max(min(MinOutdoorUnitPe, RefPHigh), RefPLow), RoutineName);
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13989]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14040]:(style),[redundantAssignment],Variable 'MinOutdoorUnitTe' is reassigned a value before the old one has been used.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 27]
```diff
@@ -14865,8 +14813,6 @@ void VRFCondenserEquipment::VRFOU_PipeLossC(
     using General::SolveRoot;
 
     int TUListNum;      // index to TU List
-    int TUIndex;        // Index to terminal unit
-    int CoilIndex;      // index to coil in terminal unit
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14869]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14868]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14869]:(style),[variableScope],The scope of the variable 'CoilIndex' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 28]
```diff
@@ -14994,8 +14940,6 @@ void VRFCondenserEquipment::VRFOU_PipeLossH(
...
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14998]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14997]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14998]:(style),[variableScope],The scope of the variable 'CoilIndex' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 29]
```diff
@@ -15120,7 +15064,6 @@ void VRFTerminalUnitEquipment::CalcVRFSuppHeatingCoil(EnergyPlusData &state,
     Real64 constexpr Acc(1.e-3); // Accuracy of solver result
 
     // local variable declaration:
-    int SolFla;              // Flag of solver, num iterations if >0, else error index
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:15123]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:15123]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 30]
```diff
@@ -14655,7 +14604,6 @@ void VRFCondenserEquipment::VRFHR_OU_HR_Mode(EnergyPlusData &state,
         Real64 Ncomp_new;
         Real64 Q_c_tot_temp;
         Real64 Q_c_OU_temp;
-        Real64 Tsuction_new;
```
Comment (rraustad) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14658]: [src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14658]:(style),[shadowVariable],Local variable 'Tsuction_new' shadows outer variable

**Comment by Myoldmopar** [comment]
Completely happy here, no reason to wait on anything, merging this.  Thanks for the continued cppcheck work @rraustad

**Pull request merged**